### ISSUE TITLE
Changing the size of the editor changes extent of interactive

### ISSUE THREAD

**Issue by linusha** [hpi-swa-lab/qinoq#1053]
This is probably undesirable, as we recently switched to use the `scale` property to change the interactives proportions while being in the editor.

**Comment by linusha** [comment]
There are two places where this is currently of importance:

- `fitBounds` on the Interactive, which is called with the desired extent when an Interactive is loaded into the editor.
- In `initializeInteractive` the following connection is created:

```
connect(this.ui.interactiveHolder, 'extent', this.interactive, 'extent');
```

Both of these places should be adopted to use e.g., converters to set the scale of the interactive.
This would allow us to achieve a situation where the extent of an interactive never gets changed due to the editor (in the worst case, a lot of scrolling and zooming would become necessary when working in the editor).
I think this is worthwhile, as it would prohibit a lot of situations that previously lead to the layout of the interactives content becoming broken.

The scale value ot be adopted should be calculated the same way that the fixed aspect ratio is applied, i.e., the height to fit is fixed.

**Comment by linusha** [comment]
Fixed in 8ec075c9a82c44580df9e0032c66627db321cc34.
